eth_bridge/ic/src/eth_proxy/src/proxy.rs

```diff
@@ -235,6 +260,29 @@ impl ToBytes for Nat {
     }
 }
 
+pub trait ToEvent {
+    fn to_cap_event(&self) -> IndefiniteEvent;
+}
+
+impl ToEvent for ClaimableMessage {
```

Comment:
```suggestion
impl ToCapEvent for ClaimableMessage {
```

Reply:
Done, didnt commit this as `commit suggestion` because im importing in different files also, so I updated all at once.